blueprints/grafana/panel_library.libsonnet

```diff
@@ -24,4 +24,23 @@
   signals: import './panels/grouped/signals.libsonnet',
   load_ramp: import './panels/grouped/load_ramp.libsonnet',
   quota_scheduler: import './panels/grouped/quota_scheduler.libsonnet',
+  postgresql: import './panels/grouped/pgsql.libsonnet',
+
+  // Stat Panels
+  active_connections: import './pgsql/active_connections.libsonnet',
```

Comment:
Why is this in a common file. Also, this is just going to get mixed up with all the panels. `active_connections` is specific to pgsql. What happens when there are other technologies we need to support?